libpod/events/config.go

```diff
@@ -33,6 +33,8 @@ type Event struct {
 	Name string `json:",omitempty"`
 	// Network is the network name in a network event
 	Network string `json:"network,omitempty"`
+	// Driver is the network driver in a network event
+	Driver string `json:",omitempty"`
```

Comment:
IMO I would not add a new field here, just throw the option in the Attributes map.

Reply:
Thanks for the pointer and reviewing, It looks better approach. I will update the code.